authentication/middleware_test.py

```diff
@@ -30,10 +29,11 @@ def test_process_exception(rf, settings):
     request.backend = backend
 
     middleware = SocialAuthExceptionRedirectMiddleware()
-    result = middleware.process_exception(request, AuthAlreadyAssociated(backend, msg))
+    error = AuthAlreadyAssociated(backend)
+    result = middleware.process_exception(request, error)
     assert result.status_code == status.HTTP_302_FOUND
     assert result.url == "{}?message={}&backend={}".format(
-        reverse("login"), urlquote(msg), backend.name
+        reverse("login"), urlquote(error.__str__()), backend.name
```

Comment:
`social-core` has been updated and the `AuthAlreadyAssociated` always returns a standard message, passing in a custom message has no effect.